lib/node_modules/@stdlib/stats/base/dists/rademacher/pmf/lib/main.js

```diff
@@ -0,0 +1,76 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MAIN //
+
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+
+/**
+* Evaluates the probability mass function (PMF) for a Rademacher distribution.
+*
+* @param {number} x - input value
+* @returns {Probability} evaluated PMF
+*
+* @example
+* var y = pmf( -1.0 );
+* // returns 0.5
+*
+* @example
+* y = pmf( 1.0 );
+* // returns 0.5
+*
+* @example
+* y = pmf( 0.4 );
+* // returns 0.0
+*
+* @example
+* y = pmf( NaN );
+* // returns NaN
+*
+* @example
+* y = pmf( 1.2 );
+* // returns NaN
+*
+* @example
+* // Invalid valid:
+* y = pmf( -2.4 );
+* // returns NaN
+*/
+function pmf( x ) {
+	if (
+		isnan( x ) ||
+		x < -1.0 ||
+		x > 1.0
+	) {
+		return NaN;
+	}
+	if ( x === 1.0 ) {
```

Comment:
Can combine L64 and L67 into a single conditional.

```suggestion
	if ( x === 1.0 || x === -1.0 ) {
```